app/uk/gov/hmrc/tradergoodsprofiles/controllers/actions/ValidateHeaderAction.scala

```diff
@@ -49,7 +50,14 @@ class ValidateHeaderAction @Inject() (datetimeService: DateTimeService)(implicit
         case Some(header) if pattern.matches(header) => Some(())
         case _                                       => None
       },
-      Some(InvalidHeaderErrorResponse(datetimeService.timestamp, "Accept header is missing or invalid").toResult)
+      Some(
+        InvalidHeaderErrorResponse(
+          uuidService.uuid,
```

Comment:
correlationId ID here has not meaning and it will only create confusion for the user or user support team as this is used to track the request downstream (to EIS), but in here we haven\t set the request to EIS yet